File: javascript/1047.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');
let initial = new Date();
let final = new Date();

initial.setHours(parseInt(lines[0]));
initial.setMinutes(parseInt(lines[1]));
initial.setSeconds(0);
final.setHours(parseInt(lines[2]));
final.setMinutes(parseInt(lines[3]));
final.setSeconds(0);

let dif = new Date(final - initial);
var result = dif.getUTCHours() == 0 && dif.getMinutes() == 0? "O JOGO DUROU 24 HORA(S) E " : "O JOGO DUROU " + dif.getUTCHours() + " HORA(S) E ";
result += dif.getUTCMinutes() + " MINUTO(S)";
console.log(result)

File: python/1047.py
import math
i=input().split(' ')
h1=int(i[0])
m1=int(i[1])
h2=int(i[2])
m2=int(i[3])

s=m1+h1*60
f=m2+h2*60
if f<=s:
    f+=24*60
c1=(f-s)/60
c2=(f-s)%60
print("O JOGO DUROU "+str(math.floor(c1))+" HORA(S) E "+str(c2)+" MINUTO(S)")